approve-merge-pr/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM node:22.13.0-alpine@sha256:f2dc6eea95f787e25f173ba9904c9d0647ab2506178c7b5b
 WORKDIR /app
 
 COPY package*.json ./
+COPY index.js ./
```

Comment:
I think we can copy index.js in the first instruction and do not need 2 distinct ones, as index.js is probably not going to be changed much